examples/reshape.cpp

```diff
@@ -0,0 +1,728 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-24, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include <cmath>
+#include <cstdlib>
+#include <cstring>
+#include <iostream>
+
+#include "RAJA/RAJA.hpp"
+#include "RAJA/util/Timer.hpp"
+
+
+#include "memoryManager.hpp"
+
+/*
+ *  Batched Matrix Multiply Example
+ *
+ *  This example performs batched matrix multiplication
+ *  for matrices of dimension 3 x 3 using two different
+ *  data layouts.
+ *
+ *  Matrices are stored in arrays A and B. Results
+ *  are stored in a third array, C.
+ *  We introduce the notation A^{e}_rc
+ *  to correspond to the matrix entry in the row, r,
+ *  column, c, of matrix, e. Below we describe the two
```

Comment:
I'm not sure what the right way to put commas here is, it feels a bit awkward to have so many commas so close together.

Reply:
Why not just simplify is like this?

We use the notation A^{e}_rc to indicate the entry in matrix e in row r at column c.